jetstream/cli.py

```diff
@@ -208,10 +221,24 @@ def convert(self, value, param, ctx):
 
 
 project_id_option = click.option(
-    "--project_id", "--project-id", default="moz-fx-data-experiments", help="Project to write to"
+    "--project_id",
+    "--project-id",
+    default="moz-fx-data-experiments",
+    help="Project to write to",
+    required=True,
```

Comment:
does required= do anything when a default is specced?

Reply:
no